### ISSUE TITLE
Dropout layer error: it gradually slowed the the training speed.

### ISSUE THREAD

**Issue by jstudy** [vlfeat/matconvnet#616]
I found this problem when training alexnet. When I trained it using batch normalization (without dropout), it run well. However, after I added dropout (use batch normalization at the same time), the training speed gradually slowed down epoch by epoch.
![image](https://cloud.githubusercontent.com/assets/18459474/16722565/fae1d7ca-4779-11e6-94f7-9d49080ced7d.png). without dropout.
![image](https://cloud.githubusercontent.com/assets/18459474/16722582/234b97aa-477a-11e6-9338-f7787dd625bf.png). with dropout.
